bin/cask

```diff
@@ -153,18 +166,38 @@ def format_version(version):
     return '.'.join(str(part) for part in version)
 
 
+def get_missing_packages():
+    """Get a report of the missing direct dependencies.
+
+    The report is a human-readable list of the any direct dependencies which are
+    required but not present. The empty string is returned if there are none.
+
+    """
+    process = subprocess.Popen([sys.executable, CASK, "missing"], stdout=subprocess.PIPE)
+    stdout, _ = process.communicate()
+
+    if(process.returncode):
+        raise Exception("Oops")
```

Comment:
Oh, well, that's too bad for `get_cask_path` ;).  Leave it out then and just do it like `get_cask_path`.  I thought that we'd have checked these things for Cask functions but I've been mistaken.

Sorry.